cmd/supernode/app/root.go

```diff
@@ -146,18 +147,26 @@ func runSuperNode() error {
 }
 
 // initLog initializes log Level and log format of daemon.
-func initLog() error {
+func initLog(dfgetLogger *logrus.Logger) error {
 	logFilePath := path.Join(options.HomeDir, "logs", "app.log")
 
 	opts := []dflog.Option{
-		dflog.WithLogFile(logFilePath),
 		dflog.WithSign(fmt.Sprintf("%d", os.Getpid())),
 		dflog.WithDebug(options.Debug),
 	}
 
-	logrus.Debugf("use log file %s", logFilePath)
+	logrus.Debugf("use app log file %s", logFilePath)
```

Comment:
So weird here. Could you please encapsulate  the code about `init a log`?

Reply:
Done